cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/AnomalyMetrics.java

```diff
@@ -58,4 +63,14 @@ public long numSelfHealingStarted() {
   public long ongoingAnomalyDurationMs() {
     return _ongoingAnomalyDurationMs;
   }
+
+  @Override
+  public String toString() {
+    return String.format("{meanTimeBetweenAnomalies:{%s:%s, %s:%s, %s:%s}, "
+                         + "meanTimeToStartFix:%s, numSelfHealingStarted:%d, ongoingAnomalyDuration=%s}",
```

Comment:
Nit: Can we update the following in (1) all hard coded Strings (e.g. including the exception messages), (2) variable names here, (3) variable names used while creating `AnomalyMetrics` object?

1. `meanTimeBetweenAnomalies` -> `meanTimeBetweenAnomaliesMs`
2. `meanTimeToStartFix` -> `meanTimeToStartFixMs`
3. `ongoingAnomalyDuration` -> `ongoingAnomalyDurationMs`

Reply:
But here we actually cast the duration to second/minute/hour/day based on duration and we have print the unit after the value.